third_party/proto/proofs.proto

```diff
@@ -10,7 +10,6 @@ enum HashOp {
     KECCAK = 3;
     RIPEMD160 = 4;
     BITCOIN = 5;  // ripemd160(sha256(x))
```

Comment:
```suggestion
    BITCOIN = 5;  // ripemd160(sha256(x))
    SHA512_256 = 6;
```

Reply:
hmm, not sure why this got added in the first place